mcs_benchmark_data/pipeline_template_metadata.py

```diff
@@ -0,0 +1,46 @@
+from typing import Optional
+from dataclasses import dataclass
+from pathlib import Path
+
+
+@dataclass(frozen=True)
+class PipelineTemplateMetadata:
+    """
+    Information about the pipeline file necessary to create a file from a template
+    """
+
+    benchmark_name: str
+    submission_name: Optional[str] = None
+    template_name: Optional[str] = None
+    dest_file_name: Optional[str] = None
+    dest_file_path_from_root: Optional[Path] = None

```

Comment:
This looks suspiciously similar to the MetadataTemplateMetadata file?

I don't really understand why these are called *Metadata.
The usual term for the data that's fed into a template is "context".